examples/jsm/loaders/GLTFLoader.js

```diff
@@ -1719,7 +1726,7 @@ var GLTFLoader = ( function () {
 	 * Requests the specified dependency asynchronously, with caching.
 	 * @param {string} type
 	 * @param {number} index
-	 * @return {Promise<Object3D|Material|Texture|AnimationClip|ArrayBuffer|Object>}
+	 * @return {Promise<Object3D|Material|THREE.Texture|AnimationClip|ArrayBuffer|Object>}
```

Comment:
this is a bit weird-looking... maybe we should just use `Foo` instead of `THREE.Foo` for all of these JSDoc comments? Doesn't need to be changed in this PR though.

Reply:
I've changed to avoid a linter warning. `modularize.js` detects `Texture` as a dependency and includes it in the import statements although the source code does not reference it.

Reply:
Yeah, your change is good, I'm just noticing that the output is not what we'd want in a source file. But that can be fixed later.